docs/how-to-guides/install-seven-add-on.md

```diff
@@ -0,0 +1,189 @@
+-   {term}`GNU make`
+-   {term}`Git`
+
+
+#### uv
+
+```{include} /_inc/_install-uv.md
+```
+
+
+#### Python
+
+{term}`uv` will download and install the required Python version for your project.
+
+
+#### nvm
+
+```{include} ../volto/_inc/_install-nvm.md
+```
+
+
+#### Node.js
+
+```{include} ../volto/_inc/_install-nodejs.md
+```
+
+3.  Enable {term}`corepack` so that Node.js will install {term}`pnpm` as a package manager.
+
+    ```shell
+    npm i -g corepack@latest && corepack enable
+    ```
+
+
+#### Make
+
+```{include} ../volto/_inc/_install-make.md
+```
+
+
+#### Git
+
+```{include} ../volto/_inc/_install-git.md
+```
+
+
+(create-project-cookieplone-generate-the-project-label)=
+
+## Generate the add-on project
+
+After satisfying the prerequisites and having activated an LTS version of Node,
+generate the add-on project.
+
+```shell
+COOKIEPLONE_REPOSITORY_TAG=seventemplate uvx cookieplone seven_addon --no-input
+```
+
+## Install the add-on project
+
+Cookieplone creates a folder with the name of the add-on, in this example, `seven-add-on`.
+
+Change your current working directory to {file}`seven-add-on`.
+
+```shell
+cd seven-add-on
+```
+
+To install the add-on setup, use the following command.
+
+```shell
+make install
+```
+
+This will take a few minutes.
+☕️
+When the process completes successfully, it will exit with no message.
+
+
+## Start Plone
+
+Plone has two servers: one for the frontend, and one for the backend.
+As such, we need to maintain two active shell sessions, one for each server, to start your Plone site.
+
+
+### Start Plone backend
+
+In the currently open session, issue the following command.
+
+```shell
+make backend-docker-start
+```
+
+The Plone backend server starts up and emits messages to the console.
+
+```console
+2024-09-25 16:47:15,699 INFO    [chameleon.config:39][MainThread] directory cache: /<path-to-project>/backend/instance/var/cache.
+2024-09-25 16:47:16,387 WARNING [ZODB.FileStorage:412][MainThread] Ignoring index for /<path-to-project>/backend/instance/var/filestorage/Data.fs
+2024-09-25 16:47:16,508 INFO    [plone.restapi.patches:16][MainThread] PATCH: Disabled ZPublisher.HTTPRequest.ZopeFieldStorage.VALUE_LIMIT. This enables file uploads larger than 1MB.
+2024-09-25 16:47:17,018 INFO    [plone.volto:23][MainThread] Aliasing collective.folderish classes to plone.volto classes.
+2024-09-25 16:47:17,760 INFO    [Zope:42][MainThread] Ready to handle requests
+Starting server in PID 20912.
+2024-09-25 16:47:17,772 INFO    [waitress:486][MainThread] Serving on http://[::1]:8080
+2024-09-25 16:47:17,772 INFO    [waitress:486][MainThread] Serving on http://127.0.0.1:8080
+```
+
+This will start a clean Plone server for development purposes so you can start developing your add-on.
+
+
+### Start Plone frontend
+
+Create a second shell session in a new window.
+Change your current working directory to {file}`seven-add-on`.
+Start the Plone frontend with the following command.
+
+```shell
+make start
```

Comment:
Are you sure? Compare to https://6.docs.plone.org/install/create-project-cookieplone.html#start-plone-frontend

```suggestion
make frontend-start
```